src/Mod/Assembly/UtilsAssembly.py

```diff
@@ -560,7 +600,7 @@ def getBomGroup(assembly):
     return bom_group
 
 
-def getJointGroup(assembly):
+def getJointGroup(assembly, create=True):
```

Comment:
Is this argument used? Also this argument seems like another function.

Reply:
Yes this is fixing a bug where a joint group would be created on the assembly deletion.
Yes creation could be a separate function but I wanted to ensure the existence on joint group when trying to access it.

Reply:
Ah, ok so this is failsafe to create if needed. It is OK then. However, the argument is not used in the function?

![image](https://github.com/user-attachments/assets/ab2cc575-ef7d-4b7c-9a7c-a2a27e7c0c2d)

Reply:
Ah well this is a very good point 😄

Reply:
Indeed now I found that I fixed the issue by another mean. And this is just a leftover of my first attempt of fixing [this bug](https://github.com/FreeCAD/FreeCAD/pull/17670).
So I will remove this now